dev/com.ibm.ws.microprofile.rest.client_fat_tck/publish/tckRunner/tck/pom-manual.xml

```diff
@@ -0,0 +1,256 @@
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License v1.0 which accompanies this distribution, 
+    and is available at 
+        http://www.eclipse.org/legal/epl-v10.html 
+    Contributors: 
+        IBM Corporation - initial API and implementation
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.ibm.ws.microprofile.rest.client</groupId>
+    <artifactId>tck.runner.tck</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>MicroProfile RestClient TCK Runner TCK Module</name>
+
+    <properties>
+        <microprofile.rest.client.version>1.0</microprofile.rest.client.version>
+        <arquillian.version>1.1.13.Final</arquillian.version>
+
+        <!-- the below is used in arquillian.xml -->
+        <!-- <wlp>${wlp}</wlp>  -->
+        <!-- <wlp>/home/andymc/mpRestClientTCK/open-liberty/dev/build.image/wlp/</wlp> -->
+        <wlp>${project.basedir}/../../../../build.image/wlp/</wlp>
+
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+
+        <defaultSuiteFiles>tck-suite.xml</defaultSuiteFiles>
+        <suiteXmlFile>${defaultSuiteFiles}</suiteXmlFile>
+        <targetDirectory>${project.basedir}/target</targetDirectory>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>${arquillian.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.eclipse.microprofile.rest.client</groupId>
+            <artifactId>microprofile-rest-client-tck</artifactId>
+            <version>${microprofile.rest.client.version}</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/../../servers/FATServer/microprofile-rest-client-tck-1.0.1.jar</systemPath>
+            <!-- <systemPath>/home/andymc/Downloads/microprofile-rest-client-tck-1.1-SNAPSHOT.jar</systemPath> -->
+        </dependency>
+
+        <dependency>
+            <groupId>org.eclipse.microprofile.rest.client</groupId>
+            <artifactId>microprofile-rest-client-api</artifactId>
+            <version>${microprofile.rest.client.version}</version>
+            <scope>system</scope>
+            <systemPath>${wlp}/dev/api/stable/com.ibm.websphere.org.eclipse.microprofile.rest.client.1.0_1.0.20.jar</systemPath>
```

Comment:
This will not survive the 1.0.20 being incremented. There is a mechanism where MvnUtils.init will 'search' for the correct full path.

Reply:
Agh - apologies - I see youhave made use of this below :-)